Products/ZenModel/migrate/updateOpenTSDBSkipUnresolvedTags.py

```diff
@@ -0,0 +1,68 @@
+##############################################################################
+#
+# Copyright (C) Zenoss, Inc. 2016, all rights reserved.
+#
+# This content is made available according to terms specified in
+# License.zenoss under the directory where your Zenoss product is installed.
+#
+##############################################################################
+
+import logging
+import Migrate
+import servicemigration as sm
+import servicemigration.service as service
+
+
+sm.require("1.0.0")
+log = logging.getLogger("zen.migrate")
+
+
+class UpdateOpenTSDBSkipUnresolvedTags(Migrate.Step):
+    """ Enable OpenTSDB option to skip unresolved tags.
+
+        This migration step iterates through services that have opentsdb.conf as an original
+        configuration file.  For each of these services both original and latest config fles
+        are checked to see to see they have the parameter 'tsd.query.skip_unresolved_tagvs'.
+        - If the parameter does not exist, then the parameter is added and set to True (enabled).
+        - If the parameter already exists, then no change is made to the config file.
+    """
+
+    version = Migrate.Version(110, 0, 0)
+    SKIP_UNRESOLVED_TAGVS = 'tsd.query.skip_unresolved_tagvs'
+
+    def cutover(self, dmd):
+        """ This method is called by the migration process and overrides the method
+            located in the parent class Migrate.Step.
+        """
+        try:
+            ctx = sm.ServiceContext()
+        except sm.ServiceMigrationError:
+            log.info("Couldn't generate service context, skipping.")
+            return
+
+        tsdbs = filter(lambda s: "opentsdb" in ctx.getServicePath(s), ctx.services)
```

Comment:
Is this lambda to get `tsdbs` just informational?  It appears it's overwritten in the next few lines.